src/App.svelte

```diff
@@ -21,10 +21,9 @@
   export let deviceUID = currentDevice.deviceUID;
 
   onMount(() => {
-    // Notehub links to a device’s dashboard using `/${deviceUID}` with no pin,
-    // and we want Notehub users to view the device’s dashboard, and not the
-    // settings page.
-    if (deviceUID && location.pathname === "/" + deviceUID && !pin) {
+    /* if the Airnote device ID is available the Airnote has most likely been 
+    set up, redirect the user automatically to their dashboard page */
+    if (deviceUID && location.pathname === "/") {
```

Comment:
This redirect is here specifically for this dashboard link in Notehub, which is why it looks for `/${deviceUID}`, and this change breaks that.

![Uploading Screen Shot 2022-08-29 at 11.11.41 AM.png…]()